honeybee/search.py

```diff
@@ -0,0 +1,41 @@
+"""Collection of methods for searching for keywords and filtering lists by keywords."""
```

Comment:
Maybe we should also add in what cases we are using this module.

Reply:
Done!